test/integration/targets/ansible-doc/test.yml

```diff
@@ -85,8 +85,8 @@
       register: result
     - assert:
         that:
-          - 'result is failed'
-          - '"ERROR! module test_no_docs_no_status missing documentation (or could not parse documentation): test_no_docs_no_status did not contain a DOCUMENTATION attribute" in result.stderr'
+          - 'result.stdout == ""'
```

Comment:
we actually want an error when no docs are provided

Reply:
That would be a different PR then.  This only takes away the error when no metadata is provided (which is what we want).

Reply:
The integration tests from line 66 on should change to 'result is failed' in that PR.